packages/flutter_tools/lib/src/ios/ios_workflow.dart

```diff
@@ -90,6 +93,17 @@ class IOSWorkflow extends DoctorValidator implements Workflow {
       ));
     }
 
+    // Python dependencies installed
+    if (hasPythonSixModule) {
+      pythonStatus = ValidationType.installed;
+    } else {
+      pythonStatus = ValidationType.missing;
+      messages.add(new ValidationMessage.error(
+        'Python installation missing module "six".\n'
+        'Install via \'sudo easy_install six\'.'
```

Comment:
Ah good point. Should we just say Install via pip install six or sudo easy_install six? Just so we don't default to the non-recommended way?